test/e2e/apimachinery/namespace.go

```diff
@@ -246,4 +248,29 @@ var _ = SIGDescribe("Namespaces [Serial]", func() {
 	ginkgo.It("should always delete fast (ALL of 100 namespaces in 150 seconds) [Feature:ComprehensiveNamespaceDraining]",
 		func() { extinguish(f, 100, 0, 150) })
 
+	ginkgo.It("should patch a Namespace", func() {
+		ginkgo.By("creating a Namespace")
+		namespaceName := "nspatchtest"
+		_, err := f.ClientSet.CoreV1().Namespaces().Create(&v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: namespaceName,
+			},
+		})
+		framework.ExpectNoError(err, "failed creating Namespace")
+
+		ginkgo.By("patching the Namespace")
+		nspatch := fmt.Sprintf(`{"metadata":{"labels":{"testLabel":"testValue"}}}`, namespaceName)
```

Comment:
```
nspatch := `{"metadata":{"labels":{"testLabel":"testValue"}}}`
```
should suffice here.

this is failing govet and staticcheck:
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/86854/pull-kubernetes-verify/1213988224351866881

Reply:
Previously the patch was also including the name of the namespace, I removed that.

Updated in https://github.com/kubernetes/kubernetes/pull/86854/commits/519ad8771b25723e13ca42cdb005fd3d4a658102